pkg/tracing/sampler/sampling.go

```diff
@@ -0,0 +1,46 @@
+package sampler
+
+import (
+	"fmt"
+
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type randGenerator interface {
+	Float64() float64
+}
+
+type RandTraceIDRatioBased struct {
```

Comment:
```suggestion
type RandomRatioBased struct {
```

This doesn't use `TraceID`, so I'd pull that from the name.